feature-libs/cart/base/components/cart-shared/cart-item-list/cart-item-list.component.spec.ts

```diff
@@ -297,6 +297,17 @@ describe('CartItemListComponent', () => {
       ).toBeDefined();
     });
 
+    it('should update controls when quantity change', () => {
```

Comment:
I believe this should be, but we might remove depending on the answer in https://github.com/SAP/spartacus/pull/15314/files#r824142667
```suggestion
    it('should update existing controls when quantity change', () => {
```